node/full.go

```diff
@@ -138,31 +140,35 @@ func newFullNode(
 
 	mp := mempoolv1.NewTxMempool(logger, llcfg.DefaultMempoolConfig(), appClient, 0)
 	mpIDs := newMempoolIDs()
+	mp.EnableTxsAvailable()
```

Comment:
can you push enabling tx available and assigning to txnAvailable in the manager, so that you can have 1 less argument to pass to `NewManager`?
```
mempool.EnableTxsAvailable()
txsAvailable = mempool.TxsAvailable()
```

Reply:
`BlockManager` does not have a reference to the mempool so this channel has to be passed as an argument